docker/README.md

```diff
@@ -0,0 +1,122 @@
+# Celestia `txsim` Docker Image Usage Guide
+
+The `txsim` binary is a tool that can be used to simulate transactions on the Celestia network. It can be used to test the performance of the Celestia network.
+This guide provides instructions on how to use the Celestia `txsim` Docker image. The `txsim` Docker image is designed to run the `txsim` binary with a variety of configurable options.
+
+# Table of Contents
```

Comment:
Thanks for the README!

markdownlint fails here: https://github.com/celestiaorg/celestia-app/actions/runs/5577999289/jobs/10191580701?pr=2052#step:4:40

```suggestion
## Table of Contents
```

Reply:
Done